README.md

```diff
@@ -13,10 +13,11 @@ Mountpoint for Amazon S3 is a simple, high-throughput file client for [mounting
 Mountpoint for Amazon S3 is **currently an alpha release and should not be used in production**. We're tracking its production readiness and future features on the [Mountpoint for Amazon S3 public roadmap](https://github.com/orgs/awslabs/projects/84). We're especially interested in early feedback on features, performance, and compatibility. Please send feedback by [opening a GitHub issue](https://github.com/awslabs/mountpoint-s3/issues/new/choose).
 
 There are some notable restrictions in this first release:
-* Mountpoint for Amazon S3 is currently **read-only**, so you won't be able to write objects back to S3 through the file system. We're working on [allowing sequential writes to new objects](https://github.com/awslabs/mountpoint-s3/issues/27) in a future release.
+
```

Comment:
I moved the readme changes to #330 and made them more extensive, so we can drop these.